src/j1cli.js

```diff
@@ -162,6 +162,11 @@ async function initializeJ1Client () {
 
 async function createEntity(j1Client, e) {
   const classLabels = Array.isArray(e.entityClass) ? e.class : [e.entityClass];
+
+  e.createdOn = e.createdOn 
```

Comment:
In the data model, it says about `createdOn`:

> The timestamp (in milliseconds since epoch) when the entity was created at the source.

I suppose that folks using the CLI should realize they need to provide a `createdOn` or else it will be set here, but I wonder if `undefined` is more appropriate since, well, it is not known "at the source"?

Reply:
it is a "convenience" feature so that i don't have to have a timestamp for each entity in the yaml/json file. agree we definitely should document this